http/headers/Priority.json

```diff
@@ -0,0 +1,40 @@
+{
+  "http": {
+    "headers": {
+      "Priority": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/HTTP/Headers/Priority",
```

Comment:
Content page does not yet exist and we don't want to generate 404 links for BCD's consumers. I will be added automatically once the docs have been published.

```suggestion
```